utils.py

```diff
@@ -38,15 +38,16 @@ def perform_rmd_conversion(stream):
         # Call R rmarkdown package from python.
         # See https://cran.r-project.org/web/packages/rmarkdown/index.html
         rmd = importr("rmarkdown")
-        rmd.render(in_file.name)
-
-        out_path = os.path.splitext(in_file.name)[0] + ".html"
+        renderedrmd = rmd.render(in_file.name)
```

Comment:
It might make the code clearer if you specified the output format (html_document) even though that is the default. I had to look up the rmarkdown documentation to find out what exactly was being rendered. This is a nitpick however, knowing what this is doing I could have assumed html was being output. Totally open to other opinions on this.